docs/readmes/basics/prerequisites.md

```diff
@@ -42,7 +42,7 @@ Development can occur from multiple OS's, where **macOS** and **Ubuntu** are **e
    pyenv install 3.8.10
    pyenv global 3.8.10
    pip3 install ansible fabric3 jsonpickle requests PyYAML
-   vagrant plugin install vagrant-vbguest vagrant-disksize
+   vagrant plugin install vagrant-vbguest vagrant-disksize vagrant-reload
```

Comment:
This is MacOS install only now. Have you also considered adopting the Linux part?

Reply:
Thanks for spotting this! I've added it to the Linux steps too, as well as `vagrant-disksize` which was also missing.